packages/bitcore-node/src/services/p2p/index.ts

```diff
@@ -45,21 +51,17 @@ export interface P2pService<Block, Transaction> {
   syncing: boolean;
 }
 
-
 export type StandardP2p = P2pService<Bitcoin.Block, Bitcoin.Transaction>;
 
-
 export type CompleteBlock<B, T> = {
-  block: B,
-  transactions: T[],
+  block: B;
+  transactions: T[];
 };
 
-
 export enum P2pEvents {
-  SYNC_COMPLETE = 'SYNC_COMPLETE',
+  SYNC_COMPLETE = 'SYNC_COMPLETE'
```

Comment:
nit: I like the trailing commas `,` they make it easier to add items.

Reply:
I don't think our linter file has that, or mine isn't respecting it